examples/vkcube/vkcube.c

```diff
@@ -1306,6 +1306,9 @@ demo_create_instance(struct demo *demo)
     VkInstanceCreateInfo instance_info = {
         .sType = VK_STRUCTURE_TYPE_INSTANCE_CREATE_INFO,
         .pNext = NULL,
+#ifdef __APPLE__
```

Comment:
I'd prefer a check to see if `inst_exts` contains "VK_KHR_portability_enumeration", and if so set the flag.